specs/paymentrequest.html

```diff
@@ -284,16 +284,78 @@
           payment app.
         </div>
         <div class="note">
-          <p>The <code>supportedMethods</code> sequence contains the <a>payment method identifiers</a>
-          for the <a>payment methods</a> that the merchant web site accepts.</p>
+
+          <p>The <code>details</code> contains the core payment request data that will be passed on to the user selected
+            <a>payment app</a>.</p>
+
+          <p>The <code>amount</code> sequence contains the amount (possibly in different currencies) that the website
+            is requesting to be paid.</p>
+
+          <p>The <code>supportedMethods</code> sequence contains the <a>payment methods</a> that the merchant web
+            site accepts. Each <a>payment method</a> is linked to one or more <a>payment method identifiers</a>.</p>
+
+          <p>The <code>identifers</code> sequence contains the <a>payment method identifiers</a> that identify the
+          <a>payment methods</a> the web site supports and for which the associated <code>data</code> is part of the
+          request.</p>
+
+          <p><code>data</code> is a <a>JSON-serializable object</a> that provides optional information that might
+          be needed by the supported <a>payment methods</a>.</p>
+
           <pre class="example highlight">
-            ["visa", "bitcoin", "bobpay.com"]
+            {
+              "amount" : [{
+                  "currency" : "USD",
+                  "value" : "55.00"
+              }],
+              "supportedMethods" : [
+                {
+                  "identifers" : ["visa", "mastercard"],
+                  "data" : {}
+                },
+                {
+                  "identifers" : ["bobpay.com"],
+                  "data" : {
```

Comment:
'data' seems too generic. Perhaps... identifierData?

Also, do we imagine being able to support identifier-specific pricing in here?